### DISCUSSION TITLE
What is the second most consumed drink in the world?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1164] [Galaxy Trivia] [answered]
**Bonus:** What is the common language of 🇪🇬, 🇱🇾, 🇲🇦 and 🇹🇳?

**Comment by lucas-0liveira** [comment]
The common language of 🇪🇬 (Egypt), 🇱🇾 (Libya), 🇲🇦 (Morocco), and 🇹🇳 (Tunisia) is Arabic. Arabic is the official language in all of these countries and is widely spoken and understood by the population.

**Comment by peterbaccaro** [comment, marked as answer]
**Tea** is the second‑most‑consumed drink in the world, after water.

The common language of Egypt, Libya, Morocco, and Tunisia is **Arabic**.

**Reply by ParanoidUser** [reply]
Glad to see you back, @peterbaccaro! 🫡
